index.html

```diff
@@ -1,19 +1,74 @@
 <!doctype html>
-
 <html lang="en">
 <head>
   <meta charset="utf-8">
-
-  <title>My Resume</title>
-
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Timothy Shores Resume</title>
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
-
   <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
-
-<body>
-  <h1>My Resume</h1>
+<body> 
+  <div class="container">
+    <nav>
+      <ul>
+        <li><a href="#">Home</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="http://timothyshores.github.io">Portfolio</a></li>
```

Comment:
That portfolio looks awesome, by the way. The total package of this resume that you built, combined with the portfolio looks really impressive. Keep building on it as we go!